modules/ROOT/pages/clustering/disaster-recovery.adoc

```diff
@@ -5,157 +5,279 @@
+The guide recovers the unavailable databases and make them fully operational, with minimal impact on the other databases in the cluster.
 However, if a database is not performing as expected for other reasons, this section cannot help.
-By following the steps outlined here, you can recover the unavailable databases and make them fully operational with minimal impact on the other databases in the cluster.
 
-[NOTE]
+[CAUTION]
 ====
-If *all* servers in a Neo4j cluster are lost in a data center failover, it is not possible to recover the current cluster.
-You have to create a new cluster and restore the databases.
-See xref:clustering/setup/deploy.adoc[Deploy a basic cluster] and xref:clustering/databases.adoc#cluster-seed[Seed a database] for more information.
+If *all* servers in a Neo4j cluster are lost in a disaster, it is not possible to recover the current cluster.
+You have to create a new cluster and restore the databases, see xref:clustering/setup/deploy.adoc[Deploy a basic cluster] and xref:clustering/databases.adoc#cluster-seed[Seed a database] for more information.
 ====
 
 == Faults in clusters
 
 Databases in clusters follow an allocation strategy.
 This means that they are allocated differently within the cluster and may also have different numbers of primaries and secondaries.
-The consequence of this is that all servers are different in which databases they are hosting.
+The consequence of this is that all servers may be different in which databases they are hosting.
 Losing a server in a cluster may cause some databases to lose a member while others are unaffected.
-Consequently, in a disaster where multiple servers go down, some databases may keep running with little to no impact, while others may lose all their allocated resources.
+Therefore, in a disaster where one or more servers go down, some databases may keep running with little to no impact, while others may lose all their allocated resources.
 
-== Guide to disaster recovery
+== Guide structure
+[NOTE]
+====
+In this guide, an _offline_ server is a server that is not running but may be restartable.
+A _lost_ server, however, is a server that is currently not running and cannot be restarted.
+A _write available_ database is able to serve writes, while a _write unavailable_ database is not.
+====
 
 There are three main steps to recovering a cluster from a disaster.
-Completing each step, regardless of the disaster scenario, is recommended to ensure the cluster is fully operational.
-
-. Ensure the `system` database is available in the cluster.
-The `system` database defines the configuration for the other databases; therefore, it is vital to ensure it is available before doing anything else.
+First, ensure the `system` database is write available.
+Then, detach any potential lost servers from the cluster and replace them by new ones.
+Finish disaster recovery by starting or continuing to manage databases and verify that they are write available.
```

Comment:
I added the list, but I don't think we should link to that part of the guide here. First of all, I think it promotes only jumping there and not considering the previous steps. Furthermore, since the others don't have similar links it makes it asymmetrical. I might add the links to all sections thought. What do you think?

Reply:
Now when I look at it, I'm wondering if it is strange to have it in list format and not mention the first step. Should I add the "Neo4j process started" section also in this overview?

Reply:
Actually, I had the same question 🙂. But then I decided it was done on purpose. I mean, here you describe three steps of disaster recovery specifically. You cannot do anything without the Neo4j process started. This makes no sense to do anything without starting Neo4j. That's why this step is beyond the described steps of disaster recovery. That was my understanding.

Reply:
I think it is not important for understanding the overall flow, but what this step wants to cover is if you have a disaster where some process might have been stopped without the user meaning to.

Reply:
In that case we don't want them to e.g. get back that the system database is write unavailable just because they haven't checked this.

Reply:
hmm, in this case there four steps:
1. Start the neo4j process on all stopped servers
2. make the 'system db write available
3. check the servers availability
4. make the databases write available again
Correct?

Reply:
Yes, maybe we should just add that in, even though the first step is very short. That is also how I presented it for field. I'll do that!

Reply:
With the links it will be more clear that this is an overview.